rory.gemspec

```diff
@@ -35,11 +35,12 @@ EOF
 
   s.add_development_dependency 'rake', '~> 10.4'
   s.add_development_dependency 'rspec', '~> 3'
+  s.add_development_dependency 'mime-types', '~> 2.6'
```

Comment:
```
capybara (2.5.0)
      mime-types (>= 1.16)
```

Capybara has a very lose requirement for mime-types version. So with this it just grabs the latest version.